service-loadbalancer/service_loadbalancer.go

```diff
@@ -326,6 +335,9 @@ func (lbc *loadBalancerController) getEndpoints(
 	// the target port are capable of service traffic for it.
 	for _, ss := range ep.Subsets {
 		for _, epPort := range ss.Ports {
+			if epPort.Protocol == api.ProtocolUDP {
```

Comment:
is this needed? i don't remember how our conversation about a similar guard around the service port in the calling function ended